test/kyberNetworkProxy.js

```diff
@@ -3464,14 +3464,14 @@ function log (string) {
 };
 
 async function makerDeposit(res, permTok, maker, ethWei, tokenTwei, kncTwei) {
-
-    await permTok.approve(res.address, tokenTwei);
-    await res.depositToken(maker, tokenTwei);
-    await KNC.approve(res.address, kncTwei);
-    await res.depositKncForFee(maker, kncTwei);
+    await permTok.approve(res.address, tokenTwei, {from: admin});
+    await res.depositToken(maker, tokenTwei, {from: admin});
```

Comment:
why file changing in this PR?

Reply:
It was causing some tests to fail, as the wrong contract was being referenced.

Reply:
Ok, I didn't check my email, will move this fix to another PR.

Reply:
I think from: admin is not required.
since admin is accounts[0] which is default account

Reply:
I've tried removing it, the permissionless orderbook reserve tests fail. I get `"Error: invalid address"`